server/plugin.go

```diff
@@ -55,7 +53,6 @@ func NewPlugin(pluginManifest model.Manifest) *Plugin {
 
 func (p *Plugin) OnActivate() (err error) {
 	mm := pluginapi.NewClient(p.API, p.Driver)
-	p.log = utils.NewPluginLogger(mm)
```

Comment:
In order to reduce the number of different loggers used, should a `PluginLogger` be used here?
```golang
log := utils.NewPluginLogger(mm, nil)
```

Once `p.conf` is setup, `log` gets reassigned `log = p.conf.NewBaseLogger()`